oteapi_dlite/strategies/parse_excel.py

```diff
@@ -153,7 +153,7 @@ def split_column_name(column):
     return name, unit
 
 
-def infer_metadata(rec: np.recarray, units: list) -> dlite.Instance:
+def infer_metadata(rec: np.recarray, units: tuple) -> dlite.Instance:
```

Comment:
Since any sequence will work, I will suggest to state that explicitly: 
```suggestion
def infer_metadata(rec: np.recarray, units: "Sequence") -> dlite.Instance:
```

This requires that we load `from typing import Sequence`.

Reply:
I agree, but this does not work together with @CasperWA's suggestion to also specify an unknown number of strings. (That results in `TypeError: Parameters to generic types must be types. Got Ellipsis.`). I will hence go for `tuple[str, ...]`.